.eslintrc.js

```diff
@@ -436,7 +436,9 @@ module.exports = {
             ],
           },
         ],
-        'import/no-cycle': ['error', { ignoreExternal: true }],
+        // TODO: Consider setting back to `ignoreExternal: true` when the expected behavior is fixed:
```

Comment:
But do we want it to be `true`? Or was it only at `true` because we thought back then it would improve performance?

Reply:
Well, as you mentioned, and they [specify in the readme](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md#ignoreexternal):
> Its value is false by default, but can be set to true for reducing total project lint time, if needed.